### ISSUE TITLE
testfilterReactions not working properly

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#983]
I missed this while reviewing, but `testfilterReactions` does not seem to be calling `filterReactions()` properly.

Running the unit test gives repeated messages of 
```
Exception TypeError: "'NoneType' object is not iterable" in 'rmgpy.species.Species.isIsomorphic' ignored
```
because the test provides `Species` objects while the method expects `Molecule` objects. The test still passes because filterReactions doesn't end up doing anything, but for the wrong reason.

It might be better to write the unit test such that there is at least one reaction that should be removed, and confirm that `filterReactions` does remove it.

@mjohnson541, do you think you could rewrite this test?

**Comment by mjohnson541** [comment]
Oh, sorry about that.  I'll get on it!

**Comment by mliu49** [comment]
Fixed by #984.
